etc/import-app

```diff
@@ -64,20 +64,21 @@ import_zip() { #given a zipfile, extract it to apps directory and return the nam
 import="$1"
 #launch a window to import an app
 if [ -z "$import" ];then
-  import="$(yad "${yadflags[@]}" --title='App importer' --width=390 \
+  import="$(yad "${yadflags[@]}" --title='App importer' --width=500 \
     --text="Import an app from somewhere else.
 Apps are saved in <b>$DIRECTORY/apps</b>.
 Put something in in the blank below.
 Examples:
+
     <b>https://github.com/Botspot/pi-apps/pull/1068</b>
     <b>1068</b>
     <b>https://link/to/app.zip</b>
-    <b>$HOME/my-app.zip</b>" \
+    <b>$HOME/my-app.zip</b>\n" \
     --form --field='' '')" || exit 0
 fi
 
-#if nothing in blank, exit now.
-[ -z "$import" ] && exit 0
+[ -z "$import" ] && yad "${yadflags[@]}" --width=310 --title="App importer" --image="${DIRECTORY}/icons/logo-64.png" --text="\n   No app has been imported." --button=Close:0 --window-icon="${DIRECTORY}/icons/logo.png"
```

Comment:
Replacing an `exit 0` with a dialog will allow the script to continue.

Reply:
Making a function that runs the dialog and then `exit 0` or making it an `if` statement are solutions.

Reply:
Fixed.